### PR TITLE
Start fleshing out vendor index view

### PR DESCRIPTION
**Pull request by ndushay**
# Why was this change made? 🤔

part of #40.  

Puts the information we have so far into the vendor index view, with the bootstrap formatting for what we have so far.

Here's where this PR gets us:

## After these changes

image

## Before these changes

image

# How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: app/views/vendors/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,31 @@
 <h1>Vendors</h1>
 
 <div id="vendors">
-  <% @vendors.each do |vendor| %>
-    <p>
-      <%= link_to vendor.folio_id, vendor_url(vendor.folio_id) %>
-    </p>
-  <% end %>
+  <div>
```
Comment (mjgiarlo) [inline app/views/vendors/index.html.erb:4]: Do we need this inner `<div>` or can the containing `<div>` hold the table?

File: app/views/vendors/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+          <th class="col-2">Can any info like next scheduled job or job freq go here?</th>
+          <th class="col-1">(Action Buttons)</th>
+        </tr>
+      </thead>
+      <tbody>
+      <% @vendors.each do |vendor| %>
+        <tr>
+          <td>
+            <%= link_to vendor.name, vendor_url(vendor.folio_id) %><br/>
+            <p>ID: <%= vendor.folio_id %>
```
Comment (mjgiarlo) [inline app/views/vendors/index.html.erb:20]: Looks like a stray, unterminated paragraph tag here.

File: spec/requests/vendors_spec.rb [hunk 3]
```diff
@@ -1,25 +1,36 @@
...
+      body = response.body
+
+      expect(body).to include("Folio's Vendor Status</th>")
+      expect(body).to include("Last Scheduled Job</th>")
+
+      expect(body).to include("/vendors/#{vendor1.folio_id}\">#{vendor1.name}</a>")
+      expect(body).to include("ID: #{vendor1.folio_id}")
+      expect(body).not_to include(vendor1.folio_code)
+
+      expect(body).to include("/vendors/#{vendor2.folio_id}\">#{vendor2.name}</a>")
```
Comment (mjgiarlo) [inline spec/requests/vendors_spec.rb:22]: I wonder if we can avoid coupling to specific HTML elements and get away with testing on the strings?

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -80,7 +80,7 @@
   # Print the 10 slowest examples and example groups at the
   # end of the spec run, to help surface which specs are running
   # particularly slow.
-  config.profile_examples = 10 unless config.files_to_run.one?
+  # config.profile_examples = 10 unless config.files_to_run.one?
```
Comment (mjgiarlo) [inline spec/spec_helper.rb:83]: I find them annoying, too. We can always turn this back on later. 👍🏻

File: app/views/vendors/index.html.erb (hunk 1 above)
Comment (ndushay) [reply app/views/vendors/index.html.erb:4]: good catch - i was flailing when i put that in.

File: spec/requests/vendors_spec.rb (hunk 3 above)
Comment (ndushay) [reply spec/requests/vendors_spec.rb:22]: I wanted to make sure we had the links.

File: app/views/vendors/index.html.erb (hunk 2 above)
Comment (ndushay) [reply app/views/vendors/index.html.erb:20]: more flailing, unremoved.

**Comment by ndushay** [comment]
@mjgiarlo ready for a re-review.

File: spec/requests/vendors_spec.rb (hunk 3 above)
Comment (mjgiarlo) [reply spec/requests/vendors_spec.rb:22]: Okie dokie, no problem.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**